### PR TITLE
Added Github Actions for `build`

### PR DESCRIPTION
**Pull request by Rawnly**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -1 +1,25 @@
-name: Build on: push: branches: - 'master' pull_request_target: branches: - 'master' jobs: build: runs-on: ubuntu-latest steps: - uses: actions/checkout@v2 - uses: actions/setup-node@v2 with: node-version: '14.x' - name: 'Install Dependencies' id: install run: yarn - name: 'Build Project' id: build run: yarn build
+name: Build
+on:
```
Comment (Rawnly) [inline .github/workflows/build.yml:2]: This will be executed on `master` and on PRs to `master`

**Review by revmischa — APPROVED** [review]

**Comment by revmischa** [comment]
Thanks!

**Comment by Rawnly** [comment]
You could also add auto-release to NPM via actions! (take a look at [Publishing NodeJS Packages](https://docs.github.com/en/actions/guides/publishing-nodejs-packages))

**Comment by Rawnly** [comment]
I've implemented version auto-bump [there](https://github.com/Rawnly/react-generated-form/blob/master/.github/workflows/publish-and-deploy.yml)

**Pull request merged**